### PR TITLE
Added aten::qr operation

### PR DESCRIPTION
**Pull request by vishwakftw**

### CODE REVIEW

File: test/cpp/test_aten_xla_tensor.cpp [hunk 1]
```diff
@@ -648,6 +648,27 @@ TEST_F(AtenXlaTensorTest, TestSVD) {
...
+        auto xla_b = at::qr(xla_a);
+        AllClose(std::get<0>(b).abs(), std::get<0>(xla_b).abs(), /*rtol=*/1e-3,
+                 /*atol=*/1e-4);
+        AllClose(std::get<1>(b).abs(), std::get<1>(xla_b).abs(), /*rtol=*/1e-3,
+                 /*atol=*/1e-4);
+      });
+    }
+  }
+}
+    }
```
Comment (dlibenzi) [inline test/cpp/test_aten_xla_tensor.cpp]: We are a bit off here :)

Comment (vishwakftw) [reply test/cpp/test_aten_xla_tensor.cpp]: Oh, sorry about that. I've fixed it.

**Comment by dlibenzi** [comment]
Please re-check everything.
We should be submitting PRs only when they pass tests locally.
At least the local CPU tests should pass, and this does not even build.

**Comment by vishwakftw** [comment]
I'm sorry about the code, I'll definitely build and test locally before sending in PRs. Thank you.

**Comment by dlibenzi** [comment]
> I'm sorry about the code, I'll definitely build and test locally before sending in PRs. Thank you.

It's all trivial stuff coming from the cur&paste from SVD, but it reduces the burden for reviewers, if submitted PRs are already green.

**Pull request merged**